Tools/CISupport/runUnittests.py

```diff
@@ -31,4 +27,15 @@ def run_unittests(test_discovery_path):
 
     path = os.path.abspath(relative_path)
     assert os.path.isdir(path), '{} is not a directory. Please specify a valid directory'.format(path)
+
```

Comment:
Not sure if we want to autoinstall even in the case when people are running this script (unit-tests) locally on their computers. 

Also would it autoinstall buildbot pip package with `--no-binary :all` flags? Usually we need that for proper buildbot installation.

Reply:
I think we _definately_ want to autoinstall when folks are running locally. One of the biggest problems I have when working on buildbot (switching machines, in particular) is that we don't have a reliable way to install buildbot and its dependencies.

`--no-binary :all:` is basically opting out of `wheel` installs, which `autoinstall` does by default. So yes, we're doing `--no-binary :all:` (except for the packages which really need to be binary, like `cryptography` and friends, but also, we don't end up installing those packages with `--no-binary :all:` in practice because they're usually already installed when installing buildbot dependencies...that being said, the "usually" here has frequently tripped me up when setting up a new machine)

Reply:
My main concern is reliability, whether autoinstall would be reliably able to install it. 
Otherwise we are going to create a mess, when autoinstall decides to ignore the package already installed on the machine, tries to install buildbot and fail. Effectively causing users problems even though they had valid buildbot version already installed.